cloud/debug/.goreleaser.yaml

```diff
@@ -0,0 +1,38 @@
+---
+project_name: aws
+release:
+  prerelease: auto
+before:
+  hooks:
+    - go work sync
+    - make predeploybin
+builds:
+  - env:
+      # - GOWORK=on
+      - CGO_ENABLED=0
+    id: aws
+    binary: aws
+    main: ./deploy/cmd
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+archives:
+  - name_template: >-
+      {{ .ProjectName }}_
+      {{- tolower .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ tolower .Arch }}{{ end }}
+    format_overrides:
+      - goos: windows
+        format: zip
+checksum:
+  name_template: "aws_checksums.txt"
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
+changelog:
+  skip: true
```

Comment:
not sure we need to release it for now, but there a quite a few `aws` references in here that could be dangerous